docs/reference/commandline/node_update.md

```diff
@@ -19,10 +19,31 @@ Update a node
 Options:
       --availability string   Availability of the node (active/pause/drain)
       --help                  Print usage
+      --label-add value       Add/update node labels (key1=value1,key2=value2)
```

Comment:
I wonder if it *should* work; for example, to set two labels, I usually need to use two `--label` flags, if this flag works different, that's inconsistent. Possibly it's how Cobra handles this, idk